src/resolvers/index.ts

```diff
@@ -43,9 +43,10 @@ export default function initResolvers({ web3 }: EthqlContext): IResolvers {
       },
     }),
 
-    Hash: new GraphQLScalarType({
-      name: 'Hash',
-      description: 'A Keccak hash, used to identify blocks and transactions',
+    Bytes32: new GraphQLScalarType({
```

Comment:
What's the reason for changing the hash to Bytes32?

Reply:
The reason is to make it a more general type, so it can be used in other places where a Bytes32 (256-bit) value is expected, e.g. log topics.